codalab/rest/bundles.py

```diff
@@ -120,6 +120,7 @@ def build_bundles_document(bundle_uuids):
     # Include permissions
     for bundle in bundles:
         json_api_include(document, BundlePermissionSchema(), bundle['group_permissions'])
+        bundle['group_permissions'] = sorted(bundle['group_permissions'], key=lambda x: x['group_name'])
```

Comment:
This won't get included in the response, this line should go above the line that precedes it.